### PR TITLE
regiodesics: new package

### PR DESCRIPTION
**Pull request by matz-e**
Used in our CI for the NSE team.

### CODE REVIEW

File: var/spack/repos/builtin/packages/regiodesics/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class Regiodesics(CMakePackage):
+    """Vector direction computation
+    """
+
+    homepage = "https://bbpteam.epfl.ch/project/spaces/display/BBPNSE/Computing+neurons+direction+vectors"
+    git = "ssh://bbpcode.epfl.ch/viz/Regiodesics"
+
+    version('master', branch='master', submodules=True)
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/regiodesics/package.py]:
```suggestion
    version('develop', branch='master', submodules=True)
```

File: var/spack/repos/builtin/packages/regiodesics/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    version('0.1.0', tag='0.1.0', submodules=True)
+
+    depends_on('cmake', type='build')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/regiodesics/package.py]: Implicit for cmake package?

File: var/spack/repos/builtin/packages/regiodesics/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/regiodesics/package.py]: https://github.com/spack/spack/pull/14915#discussion_r379210935

Comment (matz-e) [reply var/spack/repos/builtin/packages/regiodesics/package.py]: I can also just nuke that version

**Review by pramodk — APPROVED** [review]

**Comment by matz-e** [comment]
Retest this please

**Pull request merged**